packages/@ourworldindata/grapher/src/tooltip/Tooltip.tsx

```diff
@@ -1,8 +1,56 @@
 import React from "react"
+import classnames from "classnames"
 import { observable, computed, action } from "mobx"
 import { observer } from "mobx-react"
-import { Bounds } from "@ourworldindata/utils"
-import { TooltipProps, TooltipManager } from "./TooltipProps"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons"
+import { Bounds, PointVector } from "@ourworldindata/utils"
+import { TooltipProps, TooltipManager, TooltipFadeMode } from "./TooltipProps"
+export * from "./TooltipContents.js"
+
+export const TOOLTIP_FADE_DURATION = 400 // $fade-time + $fade-delay in scss
```

Comment:
Supposedly sass supports an `:export` block to expose variable values that could prevent this hard-coding, but my attempts to import the scss here and use it failed (maybe because the file is already being imported by `core/grapher.scss` and repeats are handled differently)?

Reply:
The linter doesn't like it, but this does actually change the radius of the 'current value' dot in the sparkline